packages/fields-datetime-utc/CHANGELOG.md

```diff
@@ -1,267 +0,0 @@
-# @keystonejs/fields-datetime-utc
```

Comment:
Should we keep the CHANGELOG file around until after the release so that a final version including the deprecation notice can be published?